public/app/features/dashboard/utils/panel.ts

```diff
@@ -91,6 +91,12 @@ export const sharePanel = (dashboard: DashboardModel, panel: PanelModel) => {
   });
 };
 
+export const inspectPanel = (panel: PanelModel) => {
+  console.log('TODO, inspect', panel);
+  console.log('DATA', panel.getQueryRunner());
+  alert('TODO... open modal/drawer with info');
```

Comment:
@torkelo -- what is the best way to open a modal (eventually the drawer) here?  The other examples are all angular based and I'm not sure where to look.